README.md

```diff
@@ -325,6 +325,17 @@ nc, err := nats.Connect(servers)
 // This example means 10 seconds total per backend.
 nc, err = nats.Connect(servers, nats.MaxReconnects(5), nats.ReconnectWait(2 * time.Second))
 
+// You can also add some jitter for the reconnection.
+// This call will add up to 100 milliseconds for non TLS connections and 1 second for TLS connections.
+nc, err = nats.Connect(servers, nats.ReconnectJitter(100 * time.Millisecond, time.Second))
```

Comment:
That is the default though now yes? Maybe show different values and call out the defaults here that you get with nothing added.

Reply:
You are right. I did change to:
```
// You can also add some jitter for the reconnection.
// This call will add up to 500 milliseconds for non TLS connections and 2 seconds for TLS connections.
// If not specified, the library defaults to 100 milliseconds and 1 second, respectively.
nc, err = nats.Connect(servers, nats.ReconnectJitter(500*time.Millisecond, 2*time.Second))
```

Reply:
LGTM